ReadMe - Solutions.md

```diff
@@ -0,0 +1,53 @@
+# Solution File ReadMe
```

Comment:
i feel like renaming this file to `SOLUTIONS.md` would work better for consistency, as we have files like `README.md`, `CODE_OF_CONDUCT.md`, `LICENSE.txt` and `CONTRIBUTING.md`